src/test/java/org/ambraproject/rhino/service/taxonomy/impl/TaxonomyClassificationServiceImplTest.java

```diff
@@ -254,8 +300,14 @@ public void testInvalidVectorElement() throws Exception {
   }
 
   @Test
-  @UseDataProvider("getDistinctLeafNodesTestCases")
-  public void testGetDistinctLeafNodes(int leafCount, List<WeightedTerm> input, List<WeightedTerm> expected) {
+  public void testGetDistinctLeafNodesAll() {
```

Comment:
The junit data provider runner (https://github.com/TNG/junit-dataprovider) does not work together with the junit/spring configuration system, so I had to switch to doing this manually.

Reply:
annoyance!

Reply:
annoyance!

Reply:
annoyance!

Reply:
sorry, I didn't mean you.  :)

Reply:
lol

Reply:
Honestly the data provider thing kind of sucks. I only used it because it was used in the previous testng tests and when moving to junit I wanted to change as little as possible.